helpers/termux_dependencies.py

```diff
@@ -4,19 +4,21 @@
 def InstallTermuxDependencies() : 
     CustomPrint("Installing dependencies...")
     CustomPrint("Updating Termux")
-    os.system('apt update')
-    CustomPrint("Allow storage permission for storing extraced whatsapp.ab in interal storage.")
+    os.system('pkg update')
+    CustomPrint("Allow storage permission for storing extracted whatsapp.ab in interal storage:")
     os.system('termux-setup-storage')
-    CustomPrint("Done. Installing various required components.")
-    os.system('apt install curl grep tar proot wget -y')
-    CustomPrint("Done. Installing ADB for Termux")
+    CustomPrint("Done. Installing required dependencies...")
+    os.system('pkg install curl grep tar proot wget termcolor -y')
```

Comment:
termcolor is python module and can not be installed with pkg install. Don't merge python and shell requirements together.